pkg/webhook/stats_reporter.go

```diff
@@ -18,25 +16,12 @@ const (
 )
 
 var (
-	requestCountM = stats.Int64(
-		requestCountName,
-		"The number of requests that are routed to webhook",
-		stats.UnitDimensionless)
-	responseTimeInMsecM = stats.Float64(
+	responseTimeInSecM = stats.Float64(
```

Comment:
For consistency, I would also rename this metric to "request_duration_seconds" to match the "audit_duration_seconds" metric over in the audit package.

Reply:
updated to match audit